src/schematics/src/update/update-0.13.0.ts

```diff
@@ -76,6 +76,17 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Change non-Clr prefixed classes to include the Clr prefix.
+          // We are using Cl instead of Clr in the regex for simplicity, because we export
+          // `ClarityModule,` which would be covered by this regex.
+          const reg = /import\s*{([^}]*)}\s*from\s*['"]@clr\/angular['"]/g;
+          let result = reg.exec(updated);
+          while (result !== null) {
+            const newString = result[1].replace(/\b(?!Cl)\w+\b/g, 'Clr$&');
```

Comment:
This doesn't handle the cases where we renamed like the datagrid interfaces. We need to have a list of those known name changes (mostly Datagrid interfaces, enums, but can't recall if there were others) and ensure we catch those.